lambdas/cfn-sc-actions-provider/README.md

```diff
@@ -0,0 +1,129 @@
+# Service Catalog Actions Provider CloudFormation Template Custom Resource
+
+This template and associated Lambda function and custom resource add support to AWS CloudFormation for the [AWS Service Catalog Service Actions][1]
+resource type.
+
+# Lambda
+We create the lambda in AWS to provide SC action functionality
+
+## Build Lambda
+Build the lambda using the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+
+```shell script
+sam build --build-dir lambdas/build/cfn-sc-actions-provider --base-dir lambdas/cfn-sc-actions-provider --template lambdas/cfn-sc-actions-provider/template.yaml
+```
+
+## Deploy Lambda to S3
+
+```shell script
+sam package --template-file lambdas/build/cfn-sc-actions-provider/template.yaml --s3-bucket essentials-awss3lambdaartifactsbucket-x29ftznj6pqw --output-template-file templates/cfn-sc-actions-provider.yaml
+```
```

Comment:
This is missing the step where the templates are actually copied to S3

Reply:
are you asking a question or making a suggestion?

Reply:
making a suggestion -- the directions given in the readme don't work with the deploy-templates.sh script